### ISSUE TITLE
websocket diff contains unchanged HTML attributes

### ISSUE THREAD

**Issue by ftes** [surface-ui/surface#533]
## Describe the bug
Surface transfers a larger diff via the websocket compared to heex (vanilla liveview).

Unchanged HTML attributes (e.g. `class`) are included in every diff.

## How to reproduce it
Example application: https://github.com/ftes/surface_large_diff
It contains two equivalent liveviews, one vanilla heex, one surface.

1. Run it: `mix phx.server`
2. Open the two views: `http://localhost:4000/` and `http://localhost:4000/heex`
3. Check the websocket messages produced by both

Surface websocket message:
```
["4",null,"lv:phx-FricUac5TlA4Bz7F","diff",{"1":{"d":[[" id=\"1\""," class=\"unchanged\""," class=\"also unchanged\"","2"]]}}]
```

## The behavior you expected
The surface diff should not be larger than the heex diff.
Heex websocket message:
```
["4",null,"lv:phx-FricZVm-7RAqMTxG","diff",{"0":{"d":[[" id=\"1\"","2"]]}}]
```

## Your Environment
Surface: v0.6.1
LiveView: v0.16.4
Elixir: v1.13.0-rc.0

**Comment by Malian** [comment]
@ftes Thanks for the clear reporting and the testing application. I suspect that our `for` implementation breaks something with the diff.

**Comment by msaraiva** [comment]
Fixed in b31f9657e395731233e0eae2f411009f3ef966b2.

@ftes thanks for reporting it.

Please let me know if you still find any issue.
